drivers/npmx/npmx_shell.c

```diff
@@ -2706,6 +2706,75 @@ static int cmd_vbusin_status_connected_get(const struct shell *shell, size_t arg
 	return 0;
 }
 
+static int cmd_vbusin_current_limit_get(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	if (npmx_instance == NULL) {
+		shell_error(shell, "Error: shell is not initialized.");
+		return 0;
+	}
+
+	npmx_vbusin_current_t current_limit;
+	npmx_vbusin_t *vbusin_instance = npmx_vbusin_get(npmx_instance, 0);
+	npmx_error_t err_code = npmx_vbusin_current_limit_get(vbusin_instance, &current_limit);
+
+	uint32_t current;
+	npmx_vbusin_current_convert_to_ma(current_limit, &current);
+
+	if (check_error_code(shell, err_code)) {
+		shell_print(shell, "Value: %u mA.", current);
```

Comment:
We have to consider whether the return string should start with `Value` or `Success`

Reply:
I think it would be best if `Value` would be printed, when `get` command is performed, and `Success`, when `set` command is performed. @wilu-nordic , @masz-nordic what do you think?

Reply:
Yes, that's the current standard.